vite.config.ts

```diff
@@ -3,6 +3,7 @@ import { resolve } from 'path';
 import { defineConfig } from 'vite';
 import dts from 'vite-plugin-dts';
 import { name, peerDependencies, dependencies } from './package.json';
+import ViteSvgSpriteWrapper from 'vite-svg-sprite-wrapper';
```

Comment:
Probably a sprint generator is not the best way to go, as we also need to include icons on the Next site and we can't have hardcoded routes that won't work there. Please check the other comments I left on the Icon component.